spec/factories/organisations_factory.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :organisation do
+    sequence(:id) { |index| index }
```

Comment:
I am not sure we need the id here. If you remove it all tests pass. 

Would you mind re-wording the start of your second paragraph in the commit notes.